projects/js-packages/components/components/layout/wrap/index.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.module.scss';
+
+/**
+ * JP Wrap
+ *
+ * @param {object} props - Component properties.
+ * @returns {React.Component} Wrap component.
+ */
+const Wrap = props => {
```

Comment:
`Container` looks more readable than `Wrap`. What do you think?

Reply:
I don't have a preference

Reply:
I was just making the curve close to other systems that implement that, most of them follow `Container`, `Row`, `Col`.

Ex: https://react-bootstrap.github.io/layout/grid/

Reply:
Yeah I saw that bootstrap react implementation which  also goes for Container

Reply:
should we go with Container then? (doing it)

Reply:
I vote to follow bootstrap structure, since we're doing the same.

`Container`, `Row` and `Col` 👍🏽